src/app/body/task-details/task-details.component.html

```diff
@@ -79,25 +79,19 @@ <h2>Task Details</h2>
             <div class="col-md-6 part">
                 <div class="row">
                     <div class="col-2 creationDate">
+                        <span id="headerLabel" style="cursor: default;">Creation Date</span>
                         <h5 class="pt-1">{{task.CreationDate}}</h5>
                     </div>
                     <div class="col-4 assignee">
                         <span id="headerLabel" style="cursor: default;">Assignee</span>
                         <app-suggestion-bucket [email]="task.Assignee" (selectedEmail)="selectedAssignee($event)">
                         </app-suggestion-bucket>
```

Comment:
The creator name and profile is not showing and the profile pics of users in the activity area too isnt coming

Reply:
Yeah I will check it

Reply:
But it should be resolved once the api is deployed